parm/config/config.wave

```diff
@@ -64,19 +74,24 @@ export FHINCP_WAV=`expr $DTPNT_WAV / 3600`
 # Selected output parameters (gridded)
 export OUTPARS_WAV="WND HS FP DP PHS PTP PDIR"
 
+# Options for point output (switch on/off boundary point output)
+# export DOIBP_WAV='NO'
+
+# Number of cycles to look back for restart files
+export nback=
```

Comment:
Is this from f/c-c?

Reply:
Yes, see example config.wave: /work/noaa/global/wkolczyn/save/global-workflow/expdir/example/config.wave

Reply:
Let me look closer at these, I think these are probably variables that didn't get cleaned up.  For the most part for waves, I thought we were sticking to what was in develop.

Reply:
Looks like this is something that was brought in from develop at some point but then was removed but not removed from f/c-c.  This variable is not used in develop or f/c-c so it can be removed.

Reply:
Will remove, thanks!